endpoints/conversations.py

```
@@ -0,0 +1,252 @@
+            "started_at": turn.get("started_at"),
+            "completed_at": turn.get("completed_at"),
+        }
+        chat_history.append(simplified_turn)
+
+    return chat_history
+
+
+@router.get("/conversations/{conversation_id}", responses=conversation_responses)
+async def get_conversation_endpoint_handler(
+    conversation_id: str,
+    _auth: Any = Depends(auth_dependency),
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)